README.md

```diff
@@ -1 +1,93 @@
-# perl-Ethereum-RPC-Client
\ No newline at end of file
+[![Build Status](https://travis-ci.org/binary-com/perl-Ethereum-Contract.svg?branch=master)](https://travis-ci.org/binary-com/perl-Ethereum-Contract)
+[![codecov](https://codecov.io/gh/binary-com/perl-Ethereum-Contract/branch/master/graph/badge.svg)](https://codecov.io/gh/binary-com/perl-Ethereum-Contract)
+
+# NAME
+
+perl-Ethereum-RPC
+
+# SYNOPSIS
+
+```perl
+use strict;
+use warnings;
+use Ethereum::RPC::Client;
+
+my $abi = ...;
+my $bytecode = ...;
+my $rpc_client = Ethereum::RPC::Client->new;
+
+my $coinbase = $rpc_client->eth_coinbase;
+
+my $contract = $rpc_client->contract({
+    contract_abi    => $abi,
+    from            => $from,
+    gas             => $gas,
+});
+
+my ($message, $error) = undef, undef;
+
+my ($message, $error) = $contract->deploy($bytecode)->get_contract_address(35);
```

Comment:
see https://github.com/binary-com/perl-Ethereum-RPC-Client/pull/3#discussion_r176635453 regarding the `(message, error)` return pairs.